guide/SUMMARY.md

```diff
@@ -3,7 +3,7 @@
 ### Getting started
 
 * [CommonJS and ESM](01-getting-started/commonjs-and-esm.md)
```

Comment:
I think we should keep the chapter names in the summary and the chapter headings the same, for example "Single file package configurations" in the summary vs "Single-file packages" in the heading: 
https://github.com/nodejs/package-examples/blob/2dba56b1b6caaa798036d9f6a603cb75792fd810/guide/02-single-file-package/README.md?plain=1#L5
And the same for Classic CommonJS package and Simple ESM Package.

Reply:
Hmm, the chapter names in the summary needs to be shorter because they show up in the sidebar. For example if I expand the chapter names, it can get too long:

<img width="349" height="78" alt="Screenshot 2025-09-17 at 13 28 00" src="https://github.com/user-attachments/assets/f9ef5839-0714-440c-8b41-8533d3d34288" />

Reply:
Tried my best to keep them consistent. Left the first one still a bit more elaborate (to spell out ESM).